frontend/src/_components/DynamicForm.jsx

```diff
@@ -153,6 +154,12 @@ const DynamicForm = ({
         return OpenApi;
       case 'react-component-zendesk':
         return Zendesk;
+      case 'react-component-supabase-filter':
+        return SupabaseFilter;
+      case 'react-component-supabase-sort':
```

Comment:
Can we reuse the components added in the PocketBase plugin?

Reply:
Done. Converted SupabaseColumn component to MultiColumn component which can be reused in future by passing value to type as "columns" in operations.json file